OurUmbraco/Our/SearchResultExtensions.cs

```diff
@@ -15,6 +18,25 @@ public static string GetTitle(this SearchResult result)
             return HttpContext.Current.Server.HtmlEncode(result.Fields.ContainsKey("nodeName") ? result["nodeName"] : string.Empty);
         }
 
+        public static IEnumerable<SearchResultBreadcrumbModel> GetBreadcrumbs(this SearchResult result)
```

Comment:
In the Razor view, you're calling `.ToList()` on the result of this method. Internally in the method it's already a list, so you might as well change the return type the return type to `List<SearchResultBreadcrumbModel>`. and then skip the call to `.ToList()` in the Razor might. That might help make the view look a bit more clean 😉

Reply:
I have made this change